packages/blocks/src/block-content-provider/index.js

```diff
@@ -1,20 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { Component, RawHTML } from '@wordpress/element';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { createContext, RawHTML } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { serialize } from '../api';
 
+const { Consumer, Provider } = createContext( {
```

Comment:
I think this still needs to be updated to not set the initial context as an object, but rather the noop directly.

Reply:
Right, missed this one :(